A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -737,11 +819,25 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
                 }
             }
             if (this.currentFlightPhase === FlightPhase.FLIGHT_PHASE_TAKEOFF) {
+                let n1 = this.getThrustTakeOffLimit() / 100;
+                SimVar.SetSimVarValue("AUTOPILOT THROTTLE MAX THRUST", "number", n1);
                 if (this.isAirspeedManaged()) {
                     // getCleanTakeOffSpeed is a final fallback and not truth to reality
                     const speed = isFinite(this.v2Speed) ? this.v2Speed + 10 : this.getCleanTakeOffSpeed();
                     this.setAPManagedSpeed(speed, Aircraft.A320_NEO);
                 }
+
+                //This breaks everything, not sure why (from 1.8.3 update)
+                /* let altitude = Simplane.getAltitudeAboveGround();
+                let n1 = 100;
+                if (altitude < this.thrustReductionAltitude) {
+                    n1 = this.getThrustTakeOffLimit() / 100;
+                }
+                else {
+                    n1 = this.getThrustClimbLimit() / 100;
+                }
+                SimVar.SetSimVarValue("AUTOPILOT THROTTLE MAX THRUST", "number", n1); */
```

Comment:
This simvar seems to be controlled by the thrust lever position and setting it from JS like this does nothing.

Reply:
Was that a part of Asobo's changes or something we did?

Reply:
It's part of Asobo's changes.